src/types/timezone.jl

```diff
@@ -116,9 +148,12 @@ function istimezone(str::AbstractString, mask::Class=Class(:DEFAULT))
     end
 
     # Checks against pre-compiled time zones
-    tz, class = get(_TZ_CACHE, str) do
-        nothing, Class(:NONE)
+    tz_class = get(get_tz_cache(), str, nothing)
```

Comment:
I got a performance regression without this rewrite. Not sure why, it feels it should be equivalent.

Reply:
Do you also see the performance regression with this?
```suggestion
    tz, class = get(get_tz_cache(), str, (nothing, Class(:NONE))
```